src/Juvix/Compiler/Pipeline/ImportParents.hs

```diff
@@ -0,0 +1,25 @@
+module Juvix.Compiler.Pipeline.ImportParents where
+
+import Juvix.Compiler.Concrete.Data
+import Juvix.Prelude.Base
+
+data ImportParents = ImportParents
+  { _importParents :: [TopModulePath],
+    _importParentsSeenModules :: HashSet TopModulePath
```

Comment:
is this field being used?
```
    _importParentsSeenModules :: HashSet TopModulePath
```

Reply:
No I left in this by mistake, I'll remove it.

Reply:
removed